model/Hardware/Vocabularies/CoordinateOrientationType.md

```diff
@@ -4,12 +4,14 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-CoordinateOrientationType sets the coordinate orientation. 
+CoordinateOrientationType sets the coordinate orientation.
 
 ## Description
 
-CoordinateOrientationType sets the coordinate orientation in an object. Enumerations are used to define the coordinate orientation.
-Any coordinate orientation System, that must that orientation must remain in the same order for that class’s `/Hardware/Hardware` element. 
+CoordinateOrientationType sets the coordinate orientation in an object.
+Enumerations are used to define the coordinate orientation.
+
+Any coordinate orientation system must remain in the same order for that class’s `/Hardware/Hardware` element.
```

Comment:
The original sentence is:

> Any coordinate orientation System, that must that orientation must remain in the same order for that class’s `/Hardware/Hardware` element.

Please review the "that must that orientation must remain" part.

Reply:
I will update the PR, thank you